pkg/gcv/result.go

```diff
@@ -67,6 +72,11 @@ func NewResult(
 		ConstraintViolations: make([]ConstraintViolation, len(cfResponse.Results)),
 	}
 	for idx, cfResult := range cfResponse.Results {
+		for k, _ := range cfResult.Metadata {
+			if k == ConstraintKey {
+				return nil, errors.Errorf("metadata has key %s", k)
```

Comment:
can we expand on the message or add a comment to explain why this is an error?

Reply:
added more info